src/tests/ftest/harness/basic.py

```diff
@@ -27,7 +28,34 @@ def test_always_passes_hw(self):
 
         :avocado: tags=all
         :avocado: tags=hw,large,medium,ib2,small
-        :avocado: tags=harness,harness_basic_test,test_always_passes,test_always_passes_hw
+        :avocado: tags=harness,harness_basic_test,test_always_passes_hw
         :avocado: tags=always_passes
         """
         self.test_always_passes()
+
+    def test_load_mpi(self):
+        """Simple test of apricot test code to load the openmpi module.
+
+        :avocado: tags=all
+        :avocado: tags=harness,harness_basic_test,test_load_mpi
+        :avocado: tags=load_mpi
```

Comment:
Does this need to be in pr, daily, or weekly?
Also, we generally don't use both `test_load_mpi` and `load_mpi`. Usually just `load_mpi`

Reply:
> Does this need to be in pr, daily, or weekly?

No.  At least not at this time.  It's meant to be a `Features:` (only) tag that is used when one knows one is changing code that could affect MPI stack loading.  Although it's already been identified that we need to introduce some other kind of _frequency_ tag to pick up tests that should only run with `Features:`.  But that's a different ticket.

> Also, we generally don't use both `test_load_mpi` and `load_mpi`. Usually just `load_mpi`

`test_load_mpi` is intended to be the test's globally unique identifier and `load_mpi` is a tag that can over any additional tests that we might need for testing MPI stack loading.

Reply:
> `test_load_mpi` is intended to be the test's globally unique identifier and `load_mpi` is a tag that can over any additional tests that we might need for testing MPI stack loading.

We've been trying to organize tags like:
```
:avocado: tags=<frequency>
:avocado: tags=[ci stage]
:avocado: tags=<groupings, features, etc.>
:avocado: tags=<unique_test_tag>
```